agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -201,3 +201,17 @@ class TraceDB(Model):
 
     class Config:
         collection = "traces"
+
+
+class DeploymentDB(Model):
```

Comment:
remove this

Reply:
@aakrem sorry, do you mean to remove the space or what?